src/views/registration-js/attendees/fields/birth.php

```diff
@@ -13,6 +13,7 @@
 
 $required    = isset( $field->required ) && 'on' === $field->required ? true : false;
```

Comment:
We don't need a ternary to set `true` or `false` here. We can skip that by using the logic as the bool.

Reply:
Good catch man! Thanks!